component-samples/demo/rv/rv.env

```diff
@@ -15,8 +15,6 @@ catalina_home=./target/tomcat
 rv_https_port=8443
 rv_protocol_scheme=https
 rv_ssl_keystore=/home/fdouser/certs/ssl.p12
```

Comment:
Isn't it better to keep the keystore_path variable also in creds/component.env

Reply:
This requires the key-gen script knows about the location within the docker container. This would fail if we change the location of copied file during docker COPY command.